### PR TITLE
Extend commercial ad refresh switch by 4 weeks

### PR DESCRIPTION
**Pull request by frankie297**
## What does this change?
Extends the end date of the commercial ad refresh test
## What is the value of this and can you measure success?
doesn't break the build and extends the testing period

### CODE REVIEW

**Review by scottp719 — APPROVED** [review]

File: static/src/stylesheets/module/_adslot.scss [hunk 1]
```diff
@@ -239,7 +239,7 @@
     }
 }
 
-.ad-slot--container-inline:not(.ad-slot--fluid) {
+.ad-slot--container-inline:not(.ad-slot--fluid):not(.ad-slot--gc) {
```
Comment (katebee) [inline static/src/stylesheets/module/_adslot.scss]: Ideally this change should not be part of this PR, since the two changes are not related. 😄 

This is the fix for the GoogleCard ad-slot that we also wanted to get out today, so no worries!
